### PR TITLE
Cloudnative ch1

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

- Added architecture folder
- Added architecture/cloud-native folder
- Added index.md placeholder in /architecture
- Added all book figures in /architecture/cloud-native/images
- Added full toc.yml in /architecture/cloud-native
- Added chapter 1 .md files

Additional chapters will come in separate PRs.

### CODE REVIEW

**Comment by ardalis** [comment]
Per discussion with @mairaw

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @ardalis. Left a few comments based on Acrolinx suggestions and SEO guidance.

Remember your Oxford commas 😉

**Comment by mairaw** [comment]
Also, not sure if you've used the suggestions feature before, but you can apply my suggestions in bulk via the Files changed tab.

File: docs/architecture/cloud-native/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Architecting Cloud Native .NET Applications for Azure
+description: A guide for building cloud native applications leveraging containers, microservices, and serverless features of Azure.
+author: ardalis
+ms.author: wiwagn
```
Comment (mairaw) [inline docs/architecture/cloud-native/index.md]:
```suggestion
```

**Comment by ardalis** [comment]
> Thanks @ardalis. Left a few comments based on Acrolinx suggestions and SEO guidance.
> 
> Remember your Oxford commas 😉

My co-author wrote this chapter. My review caught a bunch of Oxford comma omissions already :)

File: docs/architecture/cloud-native/azure-cloud-native-development-stack.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+Even further clouding operating system concerns is the explosive growth of containers. Containers allow applications to run independently of the operating system. "If you take a look at the way that containers have evolved, it's basically an evolution of the OS model we've had to this point," says Microsoft Azure CTO [Mark Russinovich](https://www.google.com/search?q=If+you+take+a+look+at+the+way+that+containers+have+evolved%2C+it%E2%80%99s+basically+an+evolution+of+the+OS+model+we%E2%80%99ve+had+to+this+point&rlz=1C1GCEU_enFR819US820&oq=If+you+take+a+look+at+the+way+that+containers+have+evolved%2C+it%E2%80%99s+basically+an+evolution+of+the+OS+model+we%E2%80%99ve+had+to+this+point&aqs=chrome..69i57.915j0j9&sourceid=chrome&ie=UTF-8). The operating system still exists, but its role has become further abstracted. Instead, the container orchestrator, which will we cover in detail later in the book, has become the *de-facto operating system* of choice. Developers and cloud administrators are less concerned about the operating system on which the container runs provided that the services exposed by container are reliable and can be quickly deployed.
```
[the remark this answers is not in the document]
Comment (ardalis) [reply docs/architecture/cloud-native/azure-cloud-native-development-stack.md]: @robvet

File: docs/architecture/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
```
[the remark this answers is not in the document]
Comment (ardalis) [reply docs/architecture/index.md]: should I delete it from my branch, then?

[the remark this answers is not in the document]
Comment (mairaw) [reply docs/architecture/index.md]: Yes @ardalis

**Comment by ardalis** [comment]
Can we merge this one ASAP (remember it's not going to a production branch) so I can make the ch2 PR against the updated branch?

**Comment by ardalis** [comment]
@mairaw can we merge?

**Review by mairaw — APPROVED** [review]
There are just a few comments still left. Think about those before we go live with this e-book.

**Pull request merged**